packages/perseus-editor/src/components/start-coord-settings.tsx

```diff
@@ -20,44 +25,84 @@ import type {
 type Props = PerseusGraphType & {
     range: [x: Range, y: Range];
     step: [x: number, y: number];
-    onChange: (startCoords: CollinearTuple) => void;
-};
-
-type PropsInner = {
-    type: PerseusGraphType["type"];
-    startCoords: CollinearTuple;
-    onChange: (startCoords: CollinearTuple) => void;
+    onChange: (startCoords: PerseusGraphType["startCoords"]) => void;
 };
 
-const StartCoordSettingsInner = (props: PropsInner) => {
-    const {type, startCoords, onChange} = props;
+const StartCoordSettingsInner = (props: Props) => {
+    const {type, range, step, startCoords, onChange} = props;
 
-    // Check if coords is of type CollinearTuple
     switch (type) {
+        // coords with type CollinearTuple
         case "linear":
         case "ray":
+            const linearCoords =
```

Comment:
Ignorable nit: It might be good to callout that we're either using the provided start coords, or falling back to the default ones here. It can be inferred what's going on, but it might help our future selves grok the code quicker. I've found that adding more comments helps a lot with reducing the cognitive load required to dive through a lot of our complex widget logic.